### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ignatius22**
# Features
     Navbar
     Home
     Contact

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @ignatius22! You have done a nice job designing your Restaurant Page project. However, there are some issues I'd like you to look at before moving forward. These are:
- [x] You are expected to append HTML elements to the `<div>` with `id=content` when the page is loaded (Use `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)) In your current version of the app, you did not create HTML elements. You just inserted strings that happen to be HTML.
- [x] Each `module` should export a function that creates a `<div>` element, adds the appropriate content and styles to that element, and then appends it to the DOM. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by trekab — COMMENTED** [review]
Hi @ignatius22! It appears that you didn't implement any of the suggested changes from the last review. If the feedback was unclear, please reach out to the reviewer for clarification. If you feel you need extra technical support, you can alternatively get a 1:1 session with a full-time TSE ([How to?](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-#)) We are here to help.

__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: src/contact.js
Comment (bellom) [inline src/contact.js]: we are almost there, let's use `createElement` here too.

File: src/home.js
Comment (bellom) [inline src/home.js]: Too much of comment make code look unprofessional, your code is well written and understandable.
kindly remove this comment.

File: src/home.js
Comment (bellom) [inline src/home.js]: Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

File: src/index.js
Comment (bellom) [inline src/index.js]: Too much of comment make code look unprofessional, your code is well written and understandable.
kindly remove this comment.

File: src/index.js
Comment (bellom) [inline src/index.js]: Too much of comment make code look unprofessional, your code is well written and understandable.
kindly remove this comment.

File: src/index.js
Comment (bellom) [inline src/index.js]: Too much of comment make code look unprofessional, your code is well written and understandable.
kindly remove this comment.

File: src/index.js
Comment (bellom) [inline src/index.js]: Instead of having your css inside your logic, you could use `classList` to add class then style it in your css files.
https://www.w3schools.com/jsref/prop_element_classlist.asp

File: src/index.js
Comment (bellom) [inline src/index.js]: Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

File: src/menu.js
Comment (bellom) [inline src/menu.js]: Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

File: src/navbar.js
Comment (bellom) [inline src/navbar.js]: kindly remove the comment.

File: src/navbar.js
Comment (bellom) [inline src/navbar.js]: Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/

File: .eslintrc.json
Comment (bellom) [inline .eslintrc.json]: Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/


If worst case you need to set a new rule, use inline comment;
https://eslint.org/docs/rules/no-inline-comments

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ignatius22   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi @ignatius22 :clap: :clap:
#### You have done a good job so far. However, you need to implement the required changes mentioned by the last reviewer. Specifically, you need to "separate the DOM from application logic, you can create another file for login or create another module/function for it." as instructed by @bellom 

**_Kindly separate the DOM from application logic, you can check the last reviewer comments for more details. When you are done submit for another review._**

#### _Review by Temitayo Adeleke._

File: dist/index.html
Comment (rammazzoti2000) [inline dist/index.html]: it should be main.js not bundle.js [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
## 🛑 Required Changes 🚧 👷 
Hello @ignatius22 I noticed that you chose not to implement the changes required by the previous TSE's
I suggest you do that, the changes required help you learn more as a developer and they improve the quality of your code. 

As mentioned by @trekab if you feel you need extra technical support, you can alternatively get a 1:1 session with a full-time TSE ([How to?](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-#)). We are here to help.

Among the changes already required, I have some other requirements:
### Distribution directory (dist) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
- [ ] It has a main.js file that is built by webpack, your file is called build.js, please make sure you rename it main.js
- [ ] Also the index.html links the main.js file and not the bundle.js

### Optional
All the images and styles should be placed in a separate directory (i.e. assets) in the src folder, this way you can keep your workflow structure organized.

### <em>Kindly fix the REQUIRED issues and send for another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of ```@ale[x360]``` if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @ignatius22, great job with this project :+1: However, we cannot approve the project unless you fix the comment outlined by the previous TSE. Under the `Distribution directory` `dist`

Kindly change the `bundle.js` file to `main.js` as in the screenshot below
sdc

Once this is addressed, you can submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**

KINDLY CLOSE THE PREVIOUS PR. ALTHOUGH YOU SHOULDN'T HAVE OPENED A NEW ONE.

Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**
